Manifests/ManagedPreferencesApplications/menu.nomad.login.ad.plist

```diff
@@ -1133,6 +1215,41 @@
 			<key>pfm_type</key>
 			<string>boolean</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>1.4</string>
+			<key>pfm_description</key>
+			<string>Defines which system inforation should be showed on login window, by default</string>
+			<key>pfm_name</key>
+			<string>DefaultSystemInformation</string>
+			<key>pfm_title</key>
+			<string>Default System Information</string>
+			<key>pfm_type</key>
+			<string>string</string>
+			<key>pfm_range_list</key>
+			<array>
+				<string>None</string>
+				<string>Serial</string>
+				<string>MAC</string>
+				<string>Hostname</string>
+				<string>SystemVersion</string>
+				<string>IP</string>
+			</array>
+		</dict>
```

Comment:
Should this be an array with the available options as strings?  If so, you should move the options to `pfm_subkeys`.  Additionally, since a given string would only ever be used once in the array, the `pfm_value_unique` key should be added to the array dictionary (not in the `pfm_subkeys`) with a value of `true`.

Reply:
Docs list this `boolean`, which makes no sense to me: https://gitlab.com/orchardandgrove-oss/NoMADLogin-AD/-/blob/develop/Base/Preferences.swift#L51

Reply:
Its a string. Their documentation has the information in the previous line as the case. I know its confusing.

Reply:
Im not sure about the array. I copied the same format as NotifyLogStyle

Reply:
Confirmed, this is a string. 
 Just sets the default and then cycles through the other options. https://macadmins.slack.com/archives/C88MFDLV8/p1586806542494800